harness/determined/deploy/aws/templates/efs.yaml

```diff
@@ -734,15 +734,15 @@ Resources:
                     mkdir -p /mnt/efs
                     mount -t efs ${EFS}:/ /mnt/efs
 
-              - pool_name: gpu-pool
+              - pool_name: compute-pool
```

Comment:
We need to be very careful here. How will this work around upgrading from a cluster that has experiments on `gpu-pool`? Resource pools were never designed to be removed. Does this lead to a crash if an active experiment expects `gpu-pool` to exist?

Reply:
It doesn't crash, however pending experiments error out. I've added a `breaking change` note.